doc/sphinx/source/debug.rst

```diff
@@ -135,6 +135,17 @@ The program crashes since it tries to access memory beyond buffer boundaries::
     #5  0x00007fffee90e6db in start_thread (arg=0x7fffddffe700) at pthread_create.c:463
     #6  0x00007ffff78faa3f in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95
 
+Note: printing variables (e.g. gid) could instead result in this:
+
+    (gdb) print gid
+    $1 = {{{18298392, 9223372036854775822, 0, 0}}}
+
+This happens when PoCL uses the "loops" workgroup method. The high-level overview of "loops"
```

Comment:
In fact wiloops generates a 3D loop since we do 3D work-groups only, so the printout is a 3D array (unless having less dimensions in the work-group and getting optimized).